portality/app.py

```diff
@@ -216,6 +215,25 @@ def doi_url(doi):
     return "<a href='https://doi.org/{0}'>{0}</a>".format(tendot)
 
 
+@app.template_filter('compare_value')
+def compare_value(val):
```

Comment:
I've added a unit test for testing jinja template_filters, and added a test for this function there.

Reply:
Also, I made the change you suggested, albeit with a modification, since the type of `val` is unknown, you can't just call `lower()` on it without checking you can first.